nrf_modem/doc/ug_nrf_modem_porting_os.rst

```diff
@@ -388,6 +413,25 @@ You can use it as a starting point and customize it for your OS or scheduler.
         return 0;
     }
 
+     int nrf_modem_os_mutex_init(void **mutex)
+    {
+        /* The function shall allocate and initialize a mutex and return its address
```

Comment:
```suggestion
        /* The function must allocate and initialize a mutex and return its address
```

According to the [documentation guidelines](https://nordicsemi.atlassian.net/wiki/spaces/TECHDOC/pages/120306494/Style+and+usage+guide), `must` is preferred to `shall`.